### ISSUE TITLE
Does not honor .babelrc

### ISSUE THREAD

**Issue by larron** [kmagiera/babel-watch#26]
`babel-watch` does not seem to honor my `.babelrc`

Currently:

```
{
  "presets": ["latest", "react", "stage-0"],
  "plugins": ["transform-decorators-legacy"]
}
```

Running my code with `babel-watch` and this error is thrown: `regeneratorRuntime is not defined`
While `babel-node` I am good to go

**Comment by jadsonlourenco** [comment]
Try: `npm i babel-polyfill --save-dev`, then add `import 'babel-polyfill'` to your main.js file.

**Comment by kmagiera** [comment]
@larron Thanks for reporting and @jadsonlourenco thanks for the workaround. I've tested this and looks like `babel-watch` takes `.babelrc` into account just fine. It looks like there are some issues though with runtime environments that some of the plugins require.

@larron While you can continue using the workaround, would you be able to extract a simple `main.js` file that allows you to reproduce the issue along with the `.babelrc` file you posted?

**Comment by larron** [comment]
@jadsonlourenco fix does work! Thanks for that.

@kmagiera I was able to replicate the issue with a `main.js` file containing 

```
const hey = async () => 'yo'
```

Using that same `.babelrc` above

**Comment by kmagiera** [comment]
@larron Thanks to your example code I managed to get that issue reproed.

It appears that `babel-node` automatically imports `babel-polyfill` for every process it runs. In my opinion this may lead to confusion as whenever you run your code with `babel-node` it will work fine, but when you want to run it in production it is recommended to build it first and then run with regular node. In which case you'll end up getting the same error as you have here.

Now I believe that the proper solution is what @jadsonlourenco suggested and I no longer consider it to be a workaround. I've also found an alternative solution which makes babel add `babel-polyfill` in the transpilation step and you don't need to add it manually:

```
npm install --save-dev babel-plugin-transform-runtime babel-runtime
```

Then add this to your .babelrc plugin section:

```
{
  "plugins": [
    ["transform-runtime", {
      "polyfill": false,
      "regenerator": true
    }]
  ]
}
```

I'm closing this issue, and will update troubleshooting section on the main page with information about this issue.

Thanks everyone for participating in the discussion
